### ISSUE TITLE
[BUG] Abstract properties of abstract entities do not get properly "instantiated" when extending

### ISSUE THREAD

**Issue by jacewski-bosch** [eclipse-esmf/esmf-aspect-model-editor#75]
**Describe the bug**
Abstract properties of an abstract entity that already exist on the abstract entity are not instantiated for the extending entity.

Abstract properties of the abstract entity created after extending with an extending entity populate properly.

**Additional context**

In the example below `abstractProperty1` and `abstractProperty2` have been created prior to extending `:AbstractEntityWithTwoAbstractProperties` with `:ExtendingEntity1`. They are missing from the properties list of `:ExtendingEntity1`.

 `abstractProperty3` has been created after the extension and has properly populated in the list of properties of  `:ExtendingEntity1`.

All of this is done with AME 4.4.0 but probably applies to newer versions as well.

```
@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#>.
@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#>.
@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#>.
@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#>.
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
@prefix : <urn:samm:org.eclipse.digitaltwin:1.0.0#>.

:AspectDefault a samm:Aspect;
    samm:properties (:property1);
    samm:operations ();
    samm:events ().
:property1 a samm:Property;
    samm:characteristic :Characteristic1.
:Characteristic1 a samm-c:SingleEntity;
    samm:dataType :ExtendingEntity1.
:AbstractEntityWithTwoAbstractProperties a samm:AbstractEntity;
    samm:properties (:abstractProperty1 :abstractProperty2 :abstractProperty3).
:abstractProperty1 a samm:AbstractProperty.
:abstractProperty2 a samm:AbstractProperty.
:ExtendingEntity1 a samm:Entity;
    samm:properties ([ samm:extends :abstractProperty3 ]);
    samm:extends :AbstractEntityWithTwoAbstractProperties.
:abstractProperty3 a samm:AbstractProperty.

```

**Comment by georgemoscu** [comment]
Hey @jacewski-bosch, can you give us a step-by-step scenario on how you reproduced this? Thank you!

**Comment by jacewski-bosch** [comment]
Hi @georgemoscu,
sure:

1. Create `AbstractEntity` with abstract properties (`:abstractProperty1` `:abstractProperty2`)
2. Create `ExtendingEntity1` that extends `AbstractEntity` (using the `extends` form field)
3. Tada! The `ExtendingEntity1` should now have instances of `:abstractProperty1` `:abstractProperty2` but you don't.
4. If you now add `:abstractProperty3` to `AbstractEntity` it correctly appears as an instance at `ExtendingEntity1`

Hope that helps.

**Comment by georgemoscu** [comment]
@jacewski-bosch it helps very much. Thank you for your response

**Comment by paoespinozarias** [comment]
Hi all, 

Unfortunately, I have encountered the same problem when extending abstract entities. The solution I found was to manually include in the ttl file the code corresponding to the properties that should be automatically "instantiated" when using the AME. For example, in the definition of _ExtendingEntity1_ I included something like this:
```
:ExtendingEntity1 a samm:Entity;
 samm:extends :AbstractEntityWithTwoAbstractProperties; 
 samm:properties ([samm:extends :abstractProperty1] [samm:extends :abstractProperty2]).
```

Then, when opening the file in the editor the inherited properties appear and I can specify their characteristics normally. 
However, it would be useful to fix this issue so that the AME automatically "instantiates" the properties.

Currently. I'm using the following version:
![image](https://github.com/eclipse-esmf/esmf-aspect-model-editor/assets/16767406/abcf6419-f9e5-4f69-9d7b-fb1ce3938b9b)
